docs/user-guide/cli-using-integrating-apiml.md

```diff
@@ -104,19 +201,19 @@ Use the following steps to specify a base path with Zowe V2 profiles:
 
     The format of base paths can vary based on how API ML is configured at your site.
 
-2. Using the example included in Step 1, access the API ML instance by creating or updating a [service profile](../user-guide/cli-using-using-team-profiles.md#zowe-cli-profile-types), or issuing a command, with the `--base-path` value of `ibmzosmf/api/v1`. Your service profile uses the token and credentials stored in your default base profile.
+2. Using the example in Step 1, access the API ML instance by creating or updating a [service profile](../user-guide/cli-using-using-team-profiles.md#zowe-cli-profile-types), or issuing a command, with the `--base-path` value of `ibmzosmf/api/v1`. Your service profile uses the token and credentials stored in your default base profile.
 
     To create or update a service profile with the preceding base path in a **project** team configuration file:
 
     ```
-    zowe config set "profiles.myapimlprofile.properties.basePath" "ibmzosmf/api/v1" 
+    zowe config set <profiles.myapimlprofile.properties.basePath> <ibmzosmf/api/v1> 
```

Comment:
Same comment as above - should we use placeholder names of `propertyName` and `basePath`?